lv2.yaml

```
@@ -0,0 +1,53 @@
+package:
+  name: lv2
+  epoch: 0
+  version: 1.18.10
+  description: The LV2 audio plugin specification
+  copyright:
+    - license: BSD-0-Clause
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - gtk-2.0-dev
+      - libsamplerate-dev
+      - libsndfile-dev
+      - py3-codespell
+      - py3-rdflib
+      - python3
+      - serd-tools
+      - sord-tools
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://gitlab.com/lv2/lv2.git
+      expected-commit: 0bcde338db1c63bbc503b4d1f6d7b55ed43154af
+      tag: v${{package.version}}
+
+  - uses: meson/configure
+    with:
+      opts: |-
+        -Dtests=enabled \
+        -Dplugins=enabled
```

lv2 doesn't build anymore, adding freetype fixes it

```suggestion
package:
  name: lv2
  epoch: 0
  version: 1.18.10
  description: The LV2 audio plugin specification
  copyright:
    - license: BSD-0-Clause

environment:
  contents:
    packages:
      - build-base
      - freetype-dev
      - gtk-2.0-dev
      - libsamplerate-dev
      - libsndfile-dev
      - py3-codespell
      - py3-rdflib
      - python3
      - serd-tools
      - sord-tools

pipeline:
  - uses: git-checkout
    with:
      repository: https://gitlab.com/lv2/lv2.git
      expected-commit: 0bcde338db1c63bbc503b4d1f6d7b55ed43154af
      tag: v${{package.version}}

  - uses: meson/configure
    with:
      opts: |
        -Dtests=enabled \
        -Dplugins=enabled
```